earth_enterprise/rpms/opengee-server/snippets/post-install.sh

```diff
@@ -69,6 +69,14 @@ main_postinstall()
 
     #11) done!
     service geserver start
+
+    #12) Turn off examplesearch (will be turned on by Extra, if installed).
```

Comment:
This should probably be moved up to before #6, which is where the examplesearch database is currently installed. In any case it should come before starting the server in #11. Thinking about it more, I don't think the if statement is necessary because if you install extra and then server, server was the one to actually install example search, opengee-extra just laid down the files for it. We can leave the if statement in to be safe, though.

Reply:
Postgresql needs to be running in order for the 'psql' command to execute;  thus, it is run after the geserver is started.  Moving it up after #6 causes the script to error, being unable to run 'psql'.

Reply:
Ah, OK. That makes sense.